docs/sphinx/source/api.rst

```diff
@@ -282,6 +282,14 @@ PVsyst model
 
    pvsystem.pvsyst_celltemp
 
+Functions for fitting PV models
+-------------------------------
+.. autosummary::
+   :toctree: generated/
+
+    ivtools.fit_single_diode_sandia
+    ivtools.fit_cec_sam
```

Comment:
Maybe `fit_single_diode_cec_sam` to more closely mimic the sandia function?

Reply:
I kinda like the single-diode equation (SDE) vs. single-diode model (SDM) distinction, and I don’t mind the abbreviations if they are expanded in the docstring. This leads to something succinct and descriptive like:

`fit_sde_sandia`
`fit_sdm_cec_sam`

`action_thing_[variant_]source`

I don’t know if you want to add a variant to the SDE (to cover potential variants, such as with a reverse breakdown component). Also, the variant and source could repeat in some cases, leading to some stuttering.

Reply:
@markcampanelli good suggestion for naming patterns. There are two different applications here: fitting the SD equation to an IV curve, and fitting a SD model (a set of equations) to some data (in the case of the CEC model and the SAM method, datasheet information). I changed `sde` to `single_diode` in one function name, but I don't care for `single_diode_equation` and `single_diode_model` in the names.

I'm going to open a new issue to extend the single diode documentation to lay out the terminology: single diode equation, single diode model, 5 parameters vs. model parameters.

Reply:
Works for me.